app/Resources/translations/messages.it.yml

```diff
@@ -47,7 +47,6 @@ footer:
     mozilla-ch: Mozilla Switzerland:
 ndg:
     url: https://www.nachrichtendienstgesetz.ch/
-    url2: https://ndg-referendum.ch/
```

Comment:
Plus the site it redirects to even has an italian version. Yes, it's not on topic anymore but that's being addressed elsewhere.